src/coreclr/dlls/mscoree/exports.cpp

```diff
@@ -509,3 +652,13 @@ void LogErrorToHost(const char* format, ...)
         va_end(args);
     }
 }
+
+#if defined(TARGET_ANDROID)
+void LogDebugToHost(const char* format, ...)
+{
+    va_list args;
+    va_start(args, format);
+    __android_log_vprint(ANDROID_LOG_DEBUG, ANDROID_LOGCAT_TAG, format, args);
+    va_end(args);
+}
+#endif
```

Comment:
I would follow the `LogErrorToHost` pattern here. @elinor-fung Can provide guidance on if we should be creating a new `g_errorWriter` option for debug. Perhaps this is a NOP on non-Android for now.

Reply:
I don't actually see anything using `LogDebugToHost`?